### PR TITLE
Improve docs on how to run Cosmos in Astro

### PR DESCRIPTION
**Pull request by tatiana**
Based on: https://github.com/astronomer/astronomer-cosmos/pull/778
    
Co-authored-by: Ryan Hatter <25823361+RNHTTR@users.noreply.github.com>

### CODE REVIEW

File: docs/getting_started/astro.rst [hunk 1]
```diff
@@ -5,13 +5,18 @@ Getting Started on Astro
 
 While it is possible to use Cosmos on Astro with all :ref:`Execution Modes <execution-modes>`, we recommend using the ``local`` execution mode. It's the simplest to set up and use.
 
+If you'd like to see a fully functional project to run in Astro (CLI or Cloud), `check out cosmos-demo <https://github.com/astronomer/cosmos-demo>`_.
```
Comment (pankajkoti) [inline docs/getting_started/astro.rst:8]:
```suggestion
If you'd like to see a fully functional project to run in Astro (CLI or Cloud), check out `cosmos-demo <https://github.com/astronomer/cosmos-demo>`_.
```

File: docs/getting_started/astro.rst [hunk 2]
```diff
@@ -5,13 +5,18 @@ Getting Started on Astro
...
+
+Below you can find the step-by-step to run your own dbt project within Astro.
```
Comment (pankajkoti) [inline docs/getting_started/astro.rst:10]:
```suggestion
Below you can find a step-by-step guide to run your own dbt project within Astro.
```

File: docs/getting_started/astro.rst [hunk 3]
```diff
@@ -5,13 +5,18 @@ Getting Started on Astro
...
+
 Pre-requisites
 ~~~~~~~~~~~~~~
 
 To get started, you should have:
 
 - The Astro CLI installed. You can find installation instructions `here <https://docs.astronomer.io/astro/cli/install-cli>`_.
 - An Astro CLI project. You can initialize a new project with ``astro dev init``.
+- A dbt project. The `jaffle shop example <https://github.com/dbt-labs/jaffle_shop>`_ is a good example.
```
Comment (pankajkoti) [inline docs/getting_started/astro.rst:19]: Looks like navigating to this GitHub project is getting redirected to https://github.com/dbt-labs/jaffle-shop-classic, but I guess this should be fine

**Review by pankajkoti — APPROVED** [review]

File: docs/getting_started/astro.rst (hunk 3 above)
Comment (tatiana) [reply docs/getting_started/astro.rst:19]: Thanks, @pankajkoti , I just updated to the new URL!

**Pull request merged**